diff --git a/src/gui.c b/src/gui.c
index 5ba2872..b989915 100644
--- a/src/gui.c
+++ b/src/gui.c
@@ -124,7 +124,11 @@ const char* options_title = " Options ";
 const char* stats_title = " Statistics ";
 
 /* Footer labels. */
-const char* nwipe_buttons1 = "Ctrl-C=Quit S=Start M=Method P=PRNG V=Verify R=Rounds B=Blanking-pass Space=Select";
+// lowercase to put in evidence that S is shift+S to start
+// replace quit by Refresh
+// put Start in 2nd position while on some screen it is to far
+// change letter (m) to use keys which are the same on azerty/qwerty
+const char* nwipe_buttons1 = "Ctrl-C=Refresh Space=Select S=Start t=Method p=PRNG v=Verify r=Rounds b=Blanking-pass";
 const char* nwipe_buttons2 = " J=Up K=Down Space=Select";
 const char* nwipe_buttons3 = " B=Blank screen, ctrl-c=Quit";
 
@@ -714,8 +714,8 @@ void nwipe_gui_select( int count, nwipe_context_t** c )
 				/* TODO: Explain to the user why they can't change this. */
 				break;
 
-			case 'm':
-			case 'M':
+			case 't':
+			case 'T':
 			
 				/*  Run the method dialog. */
 				nwipe_gui_method();
