diff --git a/src/version.c b/src/version.c
index df2b31e..8c23f47 100644
--- a/src/version.c
+++ b/src/version.c
@@ -14,4 +14,4 @@ Modifications to original dwipe Copyright Andy Beverley <andy@andybev.com>\n\
 This is free software; see the source for copying conditions.\n\
 There is NO warranty; not even for MERCHANTABILITY or FITNESS\n\
 FOR A PARTICULAR PURPOSE.\n";
-const char *banner         = "nwipe 0.24 (based on DBAN's dwipe - Darik's Wipe)";
+const char *banner         = "shredos - (based on nwipe)";
